build.sh

```diff
@@ -14,6 +14,7 @@ for build in ${BUILDS[@]}; do
         --build-arg go_version=${GO_VERSION} \
         --build-arg uber_prototool_version=${UBER_PROTOTOOL_VERSION} \
         --build-arg scala_pb_version=${SCALA_PB_VERSION} \
+        --build-arg mypy_version=${MYPY_VERSION} \
```

Comment:
the code analysis tool complains about missing quotes here, but I'd say it's better to stay consistent with the rest of the file

Reply:
I generally agree consistency is better but since I plan to update the project and add those missing quotes everywhere I think it's ok to start adding them here.